### PR TITLE
Add pims 0.3.3

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: recipes/pims/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  name: pims
+  version: {{ version }}
+
+source:
+  url: https://github.com/soft-matter/pims/archive/v{{ version }}.tar.gz
+  fn: pims-v{{ version }}.tar.gz
+  sha256: b764c53bfa43863f9909d7bed6fcc95008edc4e6eedb979d78a2702531282a46
+
+build:
+  number: 1
```
Comment (jakirkham) [inline recipes/pims/meta.yaml]: Please set to `0`.

**Comment by jakirkham** [comment]
One comment that needs addressing. Otherwise LGTM.

File: recipes/pims/meta.yaml (hunk 1 above)
Comment (ericdill) [inline recipes/pims/meta.yaml]: Good catch. Thanks @jakirkham

**Comment by ericdill** [comment]
Thanks @pelson :fireworks:

**Pull request merged**